### PR TITLE
Get response with decoded URI

### PR DESCRIPTION
**Pull request by kyoh86**
fix: #120

### CODE REVIEW

File: packages/service/src/shims/workspace.ts [hunk 1]
```diff
@@ -83,7 +83,7 @@ export class WorkspaceShimService extends Disposable {
 
   $getDocumentByLspUri(uri: lsp.URI): TextDocument | undefined {
```
Comment (yioneko) [inline packages/service/src/shims/workspace.ts:84]:
```suggestion
$getDocumentByLspUri(uri: lsp.URI) {
    return this._documents.get(URI.parse(uri));
}
```

Comment (yioneko) [reply packages/service/src/shims/workspace.ts:84]: Thanks for catching this! Could you try if this fix works for you? Replacing the whole method should be better for this case.

Comment (kyoh86) [reply packages/service/src/shims/workspace.ts:84]: @yioneko 
it works well.

**Comment by yioneko** [comment]
Thanks for your contribution!

**Comment by kyoh86** [comment]
@yioneko you release the latest version to npm?

**Comment by yioneko** [comment]
Done.

**Pull request merged**